Assets/Source/Entities/Behaviors/Influential.cs

```diff
@@ -1,14 +1,33 @@
 using UnityEngine;
 using System.Collections;
+using System.Collections.Generic;
 
 public class Influential : MonoBehaviour
 {
+	private static HashSet<Influential> queens = new HashSet<Influential>();
+	public static HashSet<Influential> Queens {
```

Comment:
I prefer to group properties by visibility (group all the publics together, all the privates together).